collections/cams_glc_2017.yaml

```diff
@@ -1,5 +1,4 @@
 Name: CAMS_GLC_2017
-OpenEOPID: CAMS_GLC
```

Comment:
As you put all this changes in a separate PR (which is very good) this PR should only contain the updates to the `CONTRIBUTING.md`.

Reply:
Fixed. I forgot to switch back to main first.